src/zcbenchmarks.cpp

```diff
@@ -381,7 +381,7 @@ class FakeCoinsViewDB : public CCoinsViewDB {
     bool BatchWrite(CCoinsMap &mapCoins,
                     const uint256 &hashBlock,
                     const uint256 &hashAnchor,
-                    CAnchorsMap &mapAnchors,
+                    CAnchorsSproutMap &mapSproutAnchors,
```

Comment:
Is &mapSaplingAnchors missing?

Reply:
Nice catch, nothing seems to call this though...

Reply:
Hmm... maybe we can get rid of it then in a subsequent PR.